jhu-idc/idc-isle-dc
# This fixes a few small issues: length of title/name field, Genre authorities, and EDTF formatting

This PR fixes three small issues detailed below.  I tried to do them in separate commits, which mostly worked.  

### Length of Title/Name field
https://github.com/jhu-idc/iDC-general/issues/241

This adds configuration parameters in settings.local.php to set the max length of a node's title field length and a taxonomy term's name field length.   By default they are both 256 characters. This change bumps them to title: 500 and name: 2000.  I added a new ingest test to test out a node and a taxonomy term with max data in their title/name field. 

To test: 

- Create a node with a title that is >= 500 characters and see it cut off at 500 characters. 
- Create any taxonomy term with a name that is >= 2000 characters and see it cut off at 2000 characters.

###  Genre Authority Links
https://github.com/jhu-idc/idc-isle-dc/issues/85
This simple revamps the options available in the Authority Link source dropdown. 
Adjusts an entry in the migration tests to remove a choice no longer used. 

To test: create a Genre Taxonomy term and ensure that the options available for the Authority Link Sources are 

- Art & Architecture Thesaurus
- Faceted Application of Subject Terminology
- Library of Congress Genre/Form Terms

### EDTF Field rendering
https://github.com/jhu-idc/idc-isle-dc/issues/89

The system renders the fields and hands them out to Jared and John's work via JSON.  One of the date fields had a slightly different rendering format than the other. This just makes them all render the same, like so:  

![Screen Shot 2021-04-15 at 4 01 39 PM](https://user-images.githubusercontent.com/7966375/114931200-e80a1f00-9e03-11eb-9103-964c7af33fae.png)

To test: create a repository item and fill in a bunch of dates on it.  Save and ensure that the dates show up looking similarly on the view page for the object - namely, they should all look like this YYYY-MM-DD (though if you enter modifers, they might look slightly different, but still have a four digit year, two digit month and two digit date. 

### Notes
Any changes to the ingest files are because of how they showed up after a drupal export.
